collection-scripts/gather_network_logs

```diff
@@ -14,6 +14,36 @@ function gather_multus_data {
   oc describe overlappingrangeipreservations.whereabouts.cni.cncf.io -A > "${NETWORK_LOG_PATH}"/overlappingrangeipreservations  2>&1 & PIDS=($!)
 }
 
+function gather_ovn_ipsec_data { 
+  echo "INFO: Gathering ovn-ipsec data"
+  for IPSEC_POD in ${IPSEC_PODS[@]}; do 
+    oc -n openshift-ovn-kubernetes exec "${IPSEC_POD}" -- bash -c "ovs-appctl -t ovs-monitor-ipsec tunnels/show" \
+    > "${NETWORK_LOG_PATH}"/"${IPSEC_POD}"_ovs-monitor-ipsec_tunnels-show & 
+    PIDS+=($!)
+
+    oc -n openshift-ovn-kubernetes exec "${IPSEC_POD}" -- bash -c "ipsec status" \
+    > "${NETWORK_LOG_PATH}"/"${IPSEC_POD}"_ipsec_status & 
+    PIDS+=($!)
+
+    oc -n openshift-ovn-kubernetes exec "${IPSEC_POD}" -- bash -c "certutil -L -d sql:/etc/ipsec.d" \
+    > "${NETWORK_LOG_PATH}"/"${IPSEC_POD}"_active_libreswan_certs & 
+    PIDS+=($!)
+
+    oc -n openshift-ovn-kubernetes exec "${IPSEC_POD}" -- bash -c "ovs-vsctl list Open_vSwitch" \
+    > "${NETWORK_LOG_PATH}"/"${IPSEC_POD}"_ & 
+    PIDS+=($!)
+
```

Comment:
Can you also check if ovn has ipsec enabled? I am not sure if I mentioned this to you. You can either print the entire table

`ovn-nbctl list nb_global`

or print the actual setting

`ovn-nbctl get nb_global . ipsec`

This needs to be run on ovnkube-master in the nbdb container.

Reply:
Done

Reply:
This command will probably need to run on ovnkube-master pod in the nbdb container?